tasks/move-granules/README.md

```diff
@@ -5,19 +5,38 @@
 This lambda function is responsible for moving granule files from a file staging location to their final location.
 
 ## Message Configuration
+
+For more information on configuring a Cumulus Message Adapter task, see [the Cumulus workflow input/output documentation](https://nasa.github.io/cumulus/docs/workflows/input_output).
+
 ### Config
 
+Config object fields:
+
 | field name | type | default | values | description
-| --------   | ------- | ------- | ---------- | ----------
+| ---------- | ---- | ------- | ------ | -----------
 | bucket | string | (required) | | Bucket with public/private key for decrypting CMR password
 | buckets | object | (required) | | Object specifying AWS S3 buckets used by this task
 | collection | object | (required) | | The cumulus-api collection object
 | distribution_endpoint | string | (required) | | The API distribution endpoint
-| input_granules | array\<object\> | (required) | | Array of Granule objects to construct output for Cumulus indexer
 | duplicateHandling | string | `error` | <ul><li>`error` - Throws an error on duplicates</li><li>`replace` - Replaces the existing file</li><li>`skip` - Skips the duplicate file</li><li>`version` - Adds a suffix to the existing filename to avoid a clash</li></ul> | Specifies how duplicate filenames should be handled
-| granuleIdExtraction | string | | | The regex needed for extracting granuleId from filenames
 | moveStagedFiles | boolean | `true` | | Can set to `false` to skip moving files from the staging location. Defaults to `true`.
 
+### Input
+
+Input object fields:
```

Comment:
💯  to have this, it's been a major issue explaining workflow issues to users.    We should merge approaches from #889 to generate or make this visible in the doc for search reasons /write a ticket to update all task readmes/docs/etc for PI sprint.

Reply:
Noted. Will file a ticket to improve docs with task READMEs. Probably PI material.